### DISCUSSION TITLE
x:Bind issue ?

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#9704] [Q&A]
_[From discord](https://discord.com/channels/372137812037730304/550416151172087808/1014914474764931172):_

What's the state of x:Bind right now in wasm right now? x:Bind Blah, Mode=OneWay doesn't work in wasm ?

**Comment by jeromelaban** [comment]
Can you provide a sample of what's not working? x:Bind is used a lot in many apps, you may be hitting an existing or new issue.

**Comment by roxk** [comment]
I will create a minimal repro later, but even in the uno playground `x:Bind` isn't working, whereas `Binding` works.
![image](https://user-images.githubusercontent.com/16918354/188157543-d64b67d3-0b8f-4b92-87e4-e35e99b4aa46.png)

**Reply by jeromelaban** [reply]
Thanks for the update. x:Bind in the playground cannot work at this time, as it's about code generation and `XamlReader` is used by the playground (it does not work in UWP either). You repro sample will help for sure.

**Comment by roxk** [comment]
Could not reproduce in a new project. Then, when I proceed to try to reproduce (removing the workaround) in the project that I original found the issue, it somehow works again. Can mark this thread as resolved. Thanks!
